mobile/src/main/res/values/common-styles.xml

```diff
@@ -7,15 +7,25 @@
   </style>
 
   <style name="TabStrip">
-    <item name="android:textColor">@color/show_details_tabs_text_default</item>
-    <item name="pstsTextColorSelected">@color/show_details_tabs_text_selected</item>
-    <item name="pstsIndicatorHeight">@dimen/show_details_tabs_indicator_height</item>
-    <item name="pstsIndicatorColor">@color/show_details_tabs_indicator</item>
-    <item name="pstsShouldExpand">true</item>
+    <item name="tabLayoutId">@layout/view_tab</item>
+    <item name="tabsLeftPadding">0dp</item>
+    <item name="indicatorColor">@color/tab_strip_indicator</item>
+    <item name="indicatorHeight">@dimen/tab_strip_indicator_height</item>
```

Comment:
@jpardogo hey man, tagging you so you can see what I meant properly. this is the new strip we're using - it's got 5 custom attrs (these 4 + tabsRightPadding which I didn't need here).

The tab style below is applied once (by the system) whenever a tab is inflated (note `android:textColor`'s value is a selector so we don't have to handle that either) and the tab strip will use the layout attributes from the [tab layouts](https://github.com/ataulm/wutson/pull/4/files#diff-8958147053509c27be09f320758faba5R1) so no need for the `shouldExpand` attribute.

Might be a bit buggy since it's still pretty new though :tongue: but the library is approaching a 1.0.0, and just needs a refactor and some code review.

hope this is clearer than my long-winded explanations before :)